RHEL/7/input/fixes/bash/password_require_minimum_class.sh

```diff
@@ -1,6 +1,6 @@
 grep -q minclass /etc/pam.d/system-auth
 if [ $? = "0" ]; then
-    sed --follow-symlinks -i "/pam_cracklib.so/s/minclass=[0-4]/minclass=3/" /etc/pam.d/system-auth
+    sed --follow-symlinks -i "/pam_pwquality.so/s/minclass=[0-4]/minclass=3/" /etc/pam.d/system-auth
```

Comment:
This may be a deficiency with all the `pam_pwquality` remediation scripts, but shouldn't this line change `minclass` in `/etc/security/pwquality.conf`?